satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,25 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		id
+
+	field	id				serial
+	field	user_id user.id	restrict
+	field	offer_id offer.id	restrict
+	field	referred_by user.id	restrict
+	field	credits_earned	int
+	field	credits_used	int
+	field	credits_type	int
+	field	expires_at		timestamp	(autoinsert)
+	field	created_at		timestamp	(autoinsert)
+)
+
+read one (
```

Comment:
This would be `read all` since it's very likely that one user will have multiple entry for an offer

Reply:
I noticed you added this so resolving